modules/apps/content-dashboard/content-dashboard-document-library-impl/src/main/java/com/liferay/content/dashboard/document/library/internal/item/FileEntryContentDashboardItem.java

```diff
@@ -739,11 +739,13 @@ private String _getSize(Locale locale) {
 	}
 
 	private String _getStringValue(String infoFieldName) {
-		InfoItemFieldValues infoItemFieldValues =
-			_infoItemFieldValuesProvider.getInfoItemFieldValues(_fileEntry);
+		if (_infoItemFieldValuesProvider != null) {
```

Comment:
@marianoalvarosaiz, to standardize, could we move this to a new private method  _getInfoItemFieldValuesProvider and there do the caching?

Reply:
I checked by commit @marianoalvarosaiz, then all is good. Thanks for answering so quickly!